### ISSUE TITLE
Is tutorial working?

### ISSUE THREAD

**Issue by Payback80** [notadamking/tensortrade#21]
Hi,
great job! I have installed tensortrade using git so my version is 0.0.1a4
conda dedicated venv, on ubuntu, my bug is:
`agent_spec = {
    "type": "ppo_agent",
    "step_optimizer": {
        "type": "adam",
        "learning_rate": 1e-4
    },
    "discount": 0.99,
    "likelihood_ratio_clipping": 0.2,
}
network_spec = [
    dict(type='dense', size=64, activation="tanh"),
    dict(type='dense', size=32, activation="tanh")
]
from tensortrade.strategies import TensorforceTradingStrategy
strategy = TensorforceTradingStrategy(environment=environment,
                                      agent_spec=agent_spec,
                                      network_spec=network_spec)`

returns: 

    `AttributeError                            Traceback (most recent call last)
<ipython-input-5-038eef5605bf> in <module>
      2 strategy = TensorforceTradingStrategy(environment=environment,
      3                                       agent_spec=agent_spec,
----> 4                                       network_spec=network_spec)

~/anaconda3/envs/tensortrade/lib/python3.7/site-packages/tensortrade/strategies/tensorforce_trading_strategy.py in __init__(self, environment, agent_spec, network_spec, **kwargs)
     49             self._network_spec = network_spec
     50 
---> 51             self._agent = Agent.from_spec(spec=agent_spec,
     52                                           kwargs=dict(network=network_spec,
     53                                                       states=environment.states,

    AttributeError: type object 'Agent' has no attribute 'from_spec'`

Am i missing something from the package? should i import from tensorforce? i've read they are changing some naming conventions

bye :)

**Comment by forthtemple** [comment]
I've spent a lot of time trying to get the tutorial to work but havent had any luck. I haven't come across your 'from_spec' error, but am stuck on features/feature_pipeline.py in the line:

  `          features = check_array(observation, dtype=self._dtype)`

where I get the error:

ValueError: Invalid observation frame passed to feature pipeline: Expected 2D array, got 1D array instead:
array=[10000.  10010.  10000.  10000.    285.2].
Reshape your data either using array.reshape(-1, 1) if your data has a single feature or array.reshape(1, -1) if it contains a single sample.

I then changed it to 

            features =np.array(observation) # check_array(observation, dtype=self._dtype)

which then made me a progress a little bit but then gave me a new error in the line:

`  if isinstance(self._pipeline, 'Pipeline'):`

TypeError: isinstance() arg 2 must be a type or tuple of types
